hooks/tests/test_directory_independence.sh

```
@@ -0,0 +1,41 @@
+}
+
+PROJECT_ROOT=$(find_project_root)
+echo "Detected project root: $PROJECT_ROOT"
+
+# Test from current directory (worktree_roadmap)
+echo -e "\n📝 Test 1: From current directory ($(basename $PWD))"
+echo '{"tool_name": "Write", "tool_input": {"file_path": "bad_file.py", "content": "def test(): pass"}}' | ./claude_command_scripts/check_root_files.sh | head -3
+
+# Test from subdirectory
+echo -e "\n📝 Test 2: From subdirectory (claude_command_scripts/)"
+cd claude_command_scripts
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)